lib/hmmlearn/tests/test_gmm_hmm_multisequence.py

```diff
@@ -0,0 +1,270 @@
+from hmmlearn import hmm
+import numpy as np 
+import itertools
+
+from hmmlearn.base import ConvergenceMonitor
+
+
+# Example multi-sequence data, arranged as shape
+# (n_sequences, n_samples, n_features)
+#
+# This data is a reduced subset of data_training.npy
+# from issue https://github.com/hmmlearn/hmmlearn/issues/410
+# illustrating GMMHMM fit diverging during EM iterations.
+#
+# Transformations to reduce data volume:
+# - keep only first 3 of many sequences, discard rest
+# - keep only first 50 of many samples per sequence, discard rest
+# - keep only first 4 of 17 features per sample, discard rest
+EXAMPLE_SEQUENCES_ISSUE_410_PRUNED = np.array([
```

Comment:
this input data is not a minimal example, but is a lot smaller than the data attached to the original issue.

i don't understand GMMHMM well enough to know how to construct synthetic input that might be appropriate for testing this